src/functions-reference/covariance_matrix_distributions.Rmd

```diff
@@ -110,6 +111,52 @@ lower-triangular Cholesky factor of the scale matrix `L_S`; may only be used in
 transformed data and generated quantities blocks
 `r since("2.30")`
 
+## Inverse Wishart distribution, Cholesky Parameterization (#inverse-wishart-cholesky-distribution)
+
+### Probability density function
+
+If $K \in \mathbb{N}$, $\nu \in (K-1, \infty)$, and $L_S \in
+\mathbb{R}^{K \times K}$ is lower triangular and such that $L_S L_S^{\top}$
+is positive definite,then for
+$L_W \in \mathbb{R}^{K \times K}$ is lower triangular and such that $L_WL_W^{\top}$
+is positive definite, \[
+\text{InvWishartCholesky}(L_W \mid \nu,L_S) =
```

Comment:
Inv wishart is presented as

inv_wishart(W | nu, S)

so I think the notation for inverse Wishart Cholesky should match, as in

inv_wishart_cholesky(L_W | nu, L_S)

with subscripts on both of the L variables.

Then we want to say that inv_wishart_cholesky(L_W | nu, L_S) is the density of a random variable defined as the Cholesky factorization of a draw from wishart_cholesky(nu, L_S * L_S').
```
W ~ wishart(nu, Sigma)
```

But if you want to write it all out, it needs the change of variables adjustment.

so I think for inverse Wishart with Cholesky

inv_wishart(Y | W, nu)